RiotSharp/Interfaces/IRiotApi.cs

```diff
@@ -243,7 +249,8 @@ public interface IRiotApi
         /// <param name="beginIndex">The begin index to use for fetching matches.</param>
         /// <param name="endIndex">The end index to use for fetching matches.</param>
         /// <returns>A list of Match references object.</returns>
-        MatchList GetMatchList(Region region, long summonerId, List<long> championIds = null, List<string> rankedQueues = null,
+        MatchList GetMatchList(Region region, long summonerId, List<long> championIds = null,
+            List<string> rankedQueues = null,
             List<MatchEndpoint.Enums.Season> seasons = null, DateTime? beginTime = null, DateTime? endTime = null,
             int? beginIndex = null, int? endIndex = null);
```

Comment:
could we reorganize this so that the lines are full of text? same below

Reply:
Is it okay to put this on two lines so ?

Reply:
yup I just meant that a line with only `List<string> rankedQueues = null,` feels weird